packages/react/src/oidc/vanilla/initWorker.ts

```diff
@@ -188,21 +187,6 @@ export const initWorkerAsync = async(serviceWorkerRelativeUrl, configurationName
         return await registration.unregister();
     };
 
-    registration.addEventListener('updatefound', () => {
```

Comment:
this code reloads the page if an update to a service worker is detected, is it really needed? if the user manages the service worker on his own, it probably should be on his side

Reply:
Yes, I know,

may be another property to disable this default behavior would be a nice idea.
The difficulty, is link to the time to leave request which maintain the old service worker to life and block updates.

I does not found a cleaner default behavior way. Many people does not understand well service worker so I prefer a default page realoard in production than all production page broken. 

May be a clean defaut and safer way can be found.